tools/build_cli.sh

```diff
@@ -11,7 +11,8 @@ fi
 CLI_EXE_NAME=$1
 CLI_DIR=$2
 REPO_CLI_RELATIVE_PATH=$3 # eg 'frameworks/helloworld/cli/'
-TOOLS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+source $TOOLS_DIR/init_paths.sh
```

Comment:
`TOOLS_DIR` is required when working in context of standalone repos. In context of mono-repos it works fine. Also, because we had to use `TOOLS_DIR` in one of the non-tools dir scripts, I standardized on using `TOOLS_DIR` on all other scripts for consistency.